13-hybrid-programming.Rmd

```diff
@@ -0,0 +1,68 @@
+# Hybrid Programming with SAS and R
+
+This section contains current solutions of hybrid programming.
+
+## Use {sasr}
+
+{[sasr](https://github.com/insightsengineering/sasr)} is a package built on {reticulate} so that the python package {[saspy](https://github.com/sassoftware/saspy)} can be called from R.
+It adds some simple wrappers to the {saspy} functionalities to make the syntax similar to R.
+
+Here are some simple examples of how you can use it.
+```{r eval = FALSE}
+library(sasr)
+# transfer data from R to SAS
+df2sd(mtcars)
+
+# execute SAS commands
+
+run_sas("
+  data test;
+    set work.mtcars;
+  run;
+")
+
+# transfer data back to R
+
+sd2df("test")
+
+```
+This solution is based on {[saspy](https://github.com/sassoftware/saspy)}, an official python interface to connect SAS.
```

Comment:
this is partly a repetition from above? maybe already mention this above instead?